### PR TITLE
userdbctl: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/userdbctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# userdbctl
+
+> To inspect user, groups and group memberships of the system.
```
Comment (kbdharun) [inline pages/linux/userdbctl.md:3]: How about this?

```suggestion
> Inspect user, groups and group memberships of the system.
```

File: pages/linux/userdbctl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.freedesktop.org/software/systemd/man/userdbctl.html>.
+
+- List all known users records:
```
Comment (kbdharun) [inline pages/linux/userdbctl.md:6]:
```suggestion
- List all known user records:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/userdbctl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# userdbctl
+
+> Inspect user, groups and group memberships of the system.
```
Comment (acuteenvy) [inline pages/linux/userdbctl.md:3]:
```suggestion
> Inspect users, groups and group memberships on the system.
```

File: pages/linux/userdbctl.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.freedesktop.org/software/systemd/man/userdbctl.html>.
+
+- List all known user records:
+
+`userdbctl user`
+
+- Show details of a specific user:
+
+`userdbctl user "{{username}}"`
```
Comment (acuteenvy) [inline pages/linux/userdbctl.md:12]:
```suggestion
`userdbctl user {{username}}`
```

File: pages/linux/userdbctl.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- List all known groups:
+
+`userdbctl group`
+
+- Show details of a specific group:
+
+`userdbctl group "{{groupname}}"`
```
Comment (acuteenvy) [inline pages/linux/userdbctl.md:20]:
```suggestion
`userdbctl group {{groupname}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**